drivers/spi/spi_nxp_lpspi/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 # Copyright 2024 NXP
 
 zephyr_library_sources_ifdef(CONFIG_SPI_NXP_LPSPI spi_nxp_lpspi_common.c)
-zephyr_library_sources_ifdef(CONFIG_SPI_NXP_LPSPI_CPU spi_nxp_lpspi.c)
+if(CONFIG_SPI_NXP_LPSPI_RTIO_CPU)
+	zephyr_library_sources(spi_nxp_lpspi_rtio.c)
+else()
+	zephyr_library_sources_ifdef(CONFIG_SPI_NXP_LPSPI_CPU spi_nxp_lpspi.c)
+endif()
```

Comment:
there should be no if() else() or endif() here

Reply:
Just for my own learning.. should it be the following because they are in Kconfig symbols?

zephyr_library_sources_ifdef(CONFIG_SPI_NXP_LPSPI_RTIO_CPU spi_nxp_lpspi_rtio.c)
zephyr_library_sources_ifdef(CONFIG_SPI_NXP_LPSPI_CPU spi_nxp_lpspi.c)